apis/r/src/CMakeLists.txt

```diff
@@ -0,0 +1,259 @@
+if(DEFINED ENV{LIBTILEDBSOMA_DEBUG_BUILD})
+  message(STATUS "")
+  message(STATUS "START libtiledbsoma/src/CMakeLists.txt")
+else()
+  message(STATUS "Starting TileDB-SOMA build.")
+endif()
+
+set(CMAKE_WARN_DEPRECATED OFF CACHE BOOL "" FORCE)
+
+############################################################
+# Find required dependencies
+############################################################
+
+find_package(TileDB_EP REQUIRED)
+find_package(Spdlog_EP REQUIRED)
+find_package(pybind11 REQUIRED)
```

Comment:
do we need `pybind11` here?

Reply:
Irrelevant, it's ignored. 

I copied the directory wholesale (til we find better ways to have one repo with a base library and two client language packages on top -- there are few / no precedent or best practice examples to follow).  Also note that it is in the .Rbuildignore file as R would otherwise complain.